### ISSUE TITLE
Heroku App has an Application Error

### ISSUE THREAD

**Issue by Tigstahh** [Thestickman391/Dance-Contest-Plus#1]
When trying to access the environment, it shows up with an application error.
(Sorry if this is pointless because this hasn't been update in 3 years)

**Comment by sticks-stuff** [comment]
Yeah Heroku stopped offering their free tier semi-recently so it broke this. You can self-host the server though. Didn't think anyone still used this so I haven't bothered to rehost it somewhere else. I can probably rehost it somewhere else when I get home though.
